### DISCUSSION TITLE
The Portfolio Template Error

### DISCUSSION THREAD

**Discussion by auroprashad** [webstudio-is/webstudio-community#71] [Q&A] [answered]
The default portfolio template is not opening the full edit only the geader section is visible, both in the edit page and the dashboard.[
![Screenshot (83)](https://github.com/webstudio-is/webstudio-community/assets/53395840/a66277c2-5d34-487b-9225-e9ad4ba944a4)
](URL) As shown in the screenshot the scrollbar in not visible and the sections below the header/hero section can not be accessed.
But the scroll is avaliable in other pages.

**Comment by markxtang** [comment, marked as answer]
Fixed it

![image](https://github.com/webstudio-is/webstudio-community/assets/122043657/98e8ed41-7f3f-4246-a425-6b59bb6db80c)

**Reply by auroprashad** [reply]
Thank you
